config-connector/solutions/projects/kpt/simple-project/README.md

```diff
@@ -0,0 +1,26 @@
+Simple Project
+==================================================
+
+# NAME
+
+  simple-project
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create
+  a project in an organization. Make sure your Config Connector
+  service account has the project creator role for your organization.
+
+  In order to use, replace the `${BILLING_ACCOUNT_ID?}` and `${ORG_ID?}` values
```

Comment:
You may want to add commands to set billing account ID and org ID here to better describe the usage.

Reply:
Added, now fixing syntax in next issue.